### PR TITLE
feat: adjust style complet

### LINKED ISSUE #22: Minor style adjustments
Tasks:

- Remove R from profusion logo in the first Home section
- `home-device.png` and `home-target.png` should have 350px in height, and width: auto
- Use new Contacts icons and change text to white
- Change text to white in footer
- Increase height of footer copyright bar

Contact Icons:
[icons.zip](https://github.com/profusion/pf-website/files/6361558/icons.zip)

### PR DESCRIPTION
**Pull request by GabrielBFurlani**
Closes #22

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,9 +114,8 @@ <h2 class="featurette-heading-home">
           <div class="top-space-5 bottom-space-5">
             <img
               class="img-fluid img-center"
-              width="300"
-              height="60"
-              style="height: auto"
+              width="350"
```
Comment (victor-accarini) [inline index.html:117]: The height is 350 and the width is auto

File: index.html [hunk 2]
```diff
@@ -180,8 +179,8 @@ <h2 class="featurette-heading-home">
         <div class="grid-element-7-second bottom-space-5">
           <img
             class="img-fluid img-center"
-            width="300"
-            height="60"
+            width="350"
+            height="auto"
```
Comment (victor-accarini) [inline index.html:183]: ditto

File: index.html [hunk 3]
```diff
@@ -221,7 +220,7 @@ <h2 class="featurette-heading-home">
             <br />
             <p>
               <img
-                src="https://img.icons8.com/pastel-glyph/64/000000/new-post--v2.png"
+                src="images/contact-email-icon.svg"
```
Comment (victor-accarini) [inline index.html:223]: Looking at the SVG files I realized that I forgot to minify (remove useless tags) from them, can you do it for me? https://www.svgminify.com/

**Review by victor-accarini — COMMENTED** [review]
The commit should be a `fix` commit, and the message should be like a command.

Examples:
'fix: change image styles and text colors'
'fix: reduce size of svg'
'feat: add home section'
...

**Review by victor-accarini — APPROVED** [review]

**Comment by GabrielBFurlani** [comment]
Closes #22

**Pull request merged**